configure.ac

```diff
@@ -66,8 +66,40 @@ AC_ARG_ENABLE([icb],
               [AS_VAR_SET([enable_icb], [$enableval])],
               [AS_VAR_SET([enable_icb], [no])])
 
-AM_CONDITIONAL([ICB], [test x"$enable_icb" != x"no"])
+dnl See if compiling matrix market example based on ICB
+AC_ARG_ENABLE([icb-exmm],
+              [AC_HELP_STRING([--enable-icb-exmm],
+                              [matrix market example based on ISO_C_BINDING])],
+              [
+               AS_VAR_SET([enable_icb_exmm], [$enableval])
+               AS_VAR_SET([enable_icb], [$enableval])
+              ],
+              [AS_VAR_SET([enable_icb_exmm], [no])])
+
+AM_CONDITIONAL([ICBEXMM], [test x"$enable_icb_exmm" != x"no"])
+if test x"$enable_icb_exmm" != x"no"; then
+  PKG_CHECK_MODULES([EIGEN3], [eigen3 >= 3.3.5]) dnl Go get eigen3.pc
+  AC_SUBST([EIGEN3_CFLAGS]) dnl Define EIGEN3_CFLAGS in Makefile.am
+  dnl TODO: How to pass /path/to/eigen from EIGEN3_CFLAGS to AC_CHECK_HEADERS ?...
```

Comment:
BTW, don't know how to do this. If somebody could help ?...
EIGEN3_CFLAGS="-I/path/to/eigen" => How to extract "/path/to/eigen" from EIGEN3_CFLAGS and pass it to AC_CHECK_HEADERS ?